profiles/cli.py

```diff
@@ -36,6 +38,44 @@ def run(self) -> None:
             each.clear()
 
 
+class CreateProfileCommand(Command):
+    """Allow manual creation of profiles from the commandline.
+
+    Example:
+
+    $ python manage.py create-profile --name "Test User" --email "test@test.com"
+
+    """
+    NAME = 'create-profile'
+
+    def __init__(self, profiles: Profiles):
+        super(CreateProfileCommand, self).__init__()
+        self.profiles = profiles
+
+    def get_options(self) -> List[Option]:
+        return [
+            Option('-e', '--email', dest='email'),
+            Option('-n', '--name', dest='name'),
+        ]
+
+    # pylint: disable=method-hidden,arguments-differ
+    def run(self, name, email) -> None:
+        # pylint is complaining about arguments differing from base class
+        # though this is the recommended way to define the option args in the
+        # run() definition...
+        if name and email:
+            try:
+                profile = self.profiles.get_by_email_address(email)
+            except ProfileNotFound:
+                profile = Profile(self.profiles.next_id(), Name(name.strip('"')))
```

Comment:
strange that the `strip()` is necessary?

Reply:
It was due to the `name` argument having spaces in it which would then require the value to be wrapped in quotes on the command line:

`python manage.py create-profile --name "Sean Wiseman" -- email test@test.com`

It passes `name` as `"Sean Wiseman"` hence the strip. I have since found how to let the `Option` class do this for you. Change applied.